### PR TITLE
feat(perf): Add performance score project config

### PR DESCRIPTION
**Pull request by k-fish**
### Summary
This config will be used behind a feature flag in order to opt organizations into beginning to compute performance score measurements, to be used in starfish.

### CODE REVIEW

**Comment by k-fish** [comment]
@edwardgou-sentry want to give this a look? see if the default config makes sense for what you need?

File: src/sentry/relay/config/__init__.py [hunk 1]
```diff
@@ -409,6 +409,26 @@ def _get_project_config(
             ),
         }
 
+    if features.has("organizations:performance-calculate-score-relay", project.organization):
+        config["performanceScoreConfig"] = {
+            "profiles": [
+                {
+                    "name": "Desktop",
+                    "score_components": [
```
Comment (jjbayer) [inline src/sentry/relay/config/__init__.py:417]: See relay PR.

```suggestion
                    "scoreComponents": [
```

**Review by jjbayer — COMMENTED** [review]
After merging the Relay PR and releasing the relay python bindings, and before shipping this PR, we should add a test case that calls `validate_project_config`:

https://github.com/getsentry/sentry/blob/a49fd98f4d37e4494ca350914bbcb3db21e4dc6a/tests/sentry/relay/test_config.py#L691

**Review by edwardgou-sentry — APPROVED** [review]
Hey Kevan, this looks good to me! I'll think about ttfb and include that in a separate pr if needed!

**Comment by edwardgou-sentry** [comment]
Hey @k-fish , I went ahead and just added TTFB to this pr

**Comment by k-fish** [comment]
@jjbayer Ed is going to write that test before flipping the feature flag, then we should be good to test this out on a test org before turning it on for org 1

**Comment by edwardgou-sentry** [comment]
Tests added here:
https://github.com/getsentry/sentry/pull/58788

**Pull request merged**